### DISCUSSION TITLE
GitHub copilot expired when paid for a full year

### DISCUSSION THREAD

**Discussion by osadavc** [community/community#35441] [Copilot Conversations] [answered]
Today, I got a message saying my GitHub copilot access is now ended and it directed me to a link where I have to pay for GitHub copilot but the thing is I paid for GitHub copilot like 2-3 weeks ago and I got the yearly plan. I opened a support ticket a couple of minutes ago as well.

and I am not sure if it is a coincidence or not but my Github pro plan ended today as well but pretty sure Copilot should be running without it.

Has this happened to anyone else?

**Comment by davecheney** [comment]
Where did you see this message? If it was in your ide you may not have been signed in as the account you paid for copilot with. 

<img width="475" alt="Screen Shot 2021-11-30 at 8 03 57 am" src="https://user-images.githubusercontent.com/7171/143942796-0c2b6d34-e0f4-46fe-b349-f26087b2c5ed.png">
Please try signing out, copilot will prompt you to sign back in.

**Comment by osadavc** [comment]
No actually, I am seeing the message inside GitHub dashboard's billing
page. Pretty sure that's the account I paid for because I can also see the
receipt

**Reply by davecheney** [reply]
Thank you for your reply. Please contact support who can help you. https://support.github.com/request

**Reply by osadavc** [reply]
Thanks, I did contact them yesterday but no response yet.

**Comment by andykcheng** [comment]
Same issue here.  Hope your issue is resolved soon.  I sent the request to their support 6 days agao, not even a single reply.

**Reply by osadavc** [reply]
Hope so. This is frustrating. I really love copilot but I don't have access to it for almost 2 days now. Is your case identical to mine? I mean did your GitHub pro expire recently?

**Reply by andykcheng** [reply]
Yes.  The trial period expired, so I just paid for the year. Worked for a few days, and then suddenly it is not working anymore.  Going to the copilot page in my github account, and it is asking me to pay again.  It is almost like I never paid, but my credit card and email on paying for the service does not agree.  Still no reply from github.

**Reply by osadavc** [reply]
Super frustrated right now, do you have any plan about what to do if they didn't reply ? I can even see my receipt in the payment history. is it the same with you ?
![image](https://user-images.githubusercontent.com/70420788/194851125-e8f33c6f-095c-40e5-8edd-78be81a768e8.png)

**Reply by andykcheng** [reply]
I wish I know. Yes, I have receipt and payment history too, just no one in github cares, I guess.

**Reply by vinothsmart** [reply]
Same Issue for me Paid Monthly Plan Now Showing below and buy plan again. I used plan for only couple days. 
Thank you for participating in the GitHub Copilot Technical Preview. Your extended access has ended.

Please Help on this @ximplarandy @osadavc

**Comment by osadavc** [comment, marked as answer]
They refunded me the money 🙌🏻

**Reply by andykcheng** [reply]
Lucky you! Did they send you an email or something?

**Reply by osadavc** [reply]
No, they replied to my support ticket !

**Reply by andykcheng** [reply]
Hmm. They didn't even reply to my support ticket.

**Reply by vinothsmart** [reply]
Same for No reply for received for me couple of days
